messages/commonFlags.md

```diff
@@ -0,0 +1,32 @@
+# commaWarningForTests
+
+The previous version of this command used a comma-separated list for tests. We've changed how you specify multiple tests, so if you continue using your current syntax, your tests will probably not run as you expect. 
+
+%s
+
+# flags.tests.summary
+
+Apex tests to run when --test-level is RunSpecifiedTests.
+
+# flags.tests.description
+
+If a test name contains a space, enclose it in double quotes.
+For multiple test names, use one of the following formats:
+
+- Repeat the flag for multiple test names. --tests Test1 --tests Test2 --tests "Test With Space"
+- Separate the test names with spaces --tests Test1 Test2 "Test With Space"
```

Comment:
@jshackell-sfdc small inconsistency here. One with a period, one without. Maybe a colon on both?

Reply:
Nice catch!  Yes, let's put colon for both.  Thank you!!